apsapi/__init__.py

```diff
@@ -0,0 +1,89 @@
+import httplib
+import json
+import ssl
+import urllib2
+from urlparse import urljoin
+
+
+class HTTPSClientAuthHandler(urllib2.HTTPSHandler, object):
```

Comment:
HTTPSHandler is old-style class with out object I cannot use super() in __init__()